### ISSUE TITLE
'translate-x-[20%]' applies the same CSS properties as 'translate-y-[20%]'.(cssConflict)

### ISSUE THREAD

**Issue by Nicitaa** [tailwindlabs/tailwindcss#11839]
### I want disable this warning
```css
'translate-x-[20%]' applies the same CSS properties as 'translate-y-[20%]'.(cssConflict)
```

<br/>
<hr/>
<br/>

### I have this code
src/components/Switch.tsx
```tsx
interface SwitchProps extends React.InputHTMLAttributes<HTMLInputElement> {
  isChecked: boolean
  onChange: () => void
}

export function Switch({ isChecked = false, onChange, ...props }: SwitchProps) {
  return (
    <label className="relative w-[36px] h-[20px] border-[3px] rounded-full cursor-pointer mx-2">
      <input className="hidden peer/input" type="checkbox" id="check" checked={isChecked} onChange={onChange} {...props} />
      <span className={`absolute rounded-full
      w-[10px] h-[10px] translate-x-[20%] translate-y-[20%] bg-secondary transition-all duration-500
      before:absolute before:content-['']
      peer-checked/input:translate-x-[180%]`} />
    </label>
  )
}
```
error here -  translate-x-[20%] translate-y-[20%]

<br/>
<hr/>
<br/>

### Reproduce example - [codesandbox](https://codesandbox.io/p/sandbox/dark-mode-w2s4gd?file=%2Fsrc%2FApp.tsx%3A28%2C50)

<br/>
<br/>
<hr/>

### I tried

1. Delete node_modules and package.json.lock and `yarn`

2. Put this setting in my settings.json (I use vscode)
```json
	"tailwindCSS.experimentalCSSValidation": false,
```
I got this error
```
Unknown Configuration Setting
```

**What version of Tailwind CSS are you using?**
```
  "dependencies": {
      "tailwind-merge": "^1.14.0",
      },
        "devDependencies": {
         "autoprefixer": "^10.4.15",
        "tailwindcss": "^3.3.3",
        "postcss": "^8.4.28",
 ```

**What build tool (or framework if it abstracts the build tool) are you using?**
React + Vite + TypeScript + tailwind + zustand

<details>

<summary>package.json</summary>

```json
{
  "name": "codesandbox-dark-mode",
  "private": true,
  "version": "0.0.0",
  "type": "module",
  "scripts": {
    "dev": "vite --port 8000",
    "build": "tsc && vite build",
    "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
    "preview": "vite preview"
  },
  "dependencies": {
    "@types/node": "^20.5.0",
    "class-variance-authority": "^0.7.0",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-icons": "^4.10.1",
    "react-router": "^6.12.1",
    "react-router-dom": "^6.12.1",
    "tailwind-merge": "^1.14.0",
    "zustand": "^4.4.1"
  },
  "devDependencies": {
    "@types/react": "^18.2.7",
    "@types/react-dom": "^18.2.4",
    "@typescript-eslint/eslint-plugin": "^5.57.1",
    "@typescript-eslint/parser": "^5.57.1",
    "@vitejs/plugin-react-swc": "^3.0.0",
    "autoprefixer": "^10.4.15",
    "eslint": "^8.38.0",
    "eslint-plugin-react-hooks": "^4.6.0",
    "eslint-plugin-react-refresh": "^0.3.4",
    "postcss": "^8.4.28",
    "tailwindcss": "^3.3.3",
    "typescript": "^5.0.2",
    "vite": "^4.3.2"
  }
}

```

</details>

**What version of Node.js are you using?**
v16.17.0

**What browser are you using?**
Chrome

**What operating system are you using?**
Kali linux (Is it take matter?)

**Comment by bradlc** [comment]
Duplicate of https://github.com/tailwindlabs/tailwindcss-intellisense/issues/841

**Comment by Nicitaa** [comment]
> Duplicate of [tailwindlabs/tailwindcss-intellisense#841](https://github.com/tailwindlabs/tailwindcss-intellisense/issues/841)

Thank you for quick fix in v0.9.13

I would change
**What version of Tailwind CSS are you using?**
To
**What version of Tailwind CSS IntelliSence are you using?**
